### ISSUE TITLE
Unable to bulid aircraft_music bank due to those missing files. idk why

### ISSUE THREAD

**Issue by Iwan1907** [GaijinEntertainment/fmod_studio_warthunder_for_modders#59]
Idk why this error box popups. When I was operating, I didn't have it before. Right now  I cannot do anything about it. There are any such files in repository.. Have somegody got an idea about it?
![error](https://user-images.githubusercontent.com/77158908/104025686-48fed700-51c5-11eb-92f0-4fec56ea9bff.png)

**Comment by RadonSound** [comment]
Hello,
have you checked if the  1_japan_wake  event is empty? 
![Screenshot (562)](https://user-images.githubusercontent.com/67481432/104033916-7a30d480-51d0-11eb-8453-d8bf2abf8ccc.png)

**Comment by Uchuujin01** [comment]
@Iwan1907 hello! Like @RadonSound said, be sure all music events are empty (doesn't contain missing music instruments).

**Comment by OXEPU3M** [comment]
@Uchuujin01 I tryed put in only menu music. And it didnt work.
So can you tell why "menu and new menu" events are need for?
Is it possible to set up a single theme for menu instead 8 national?

**Comment by Uchuujin01** [comment]
I'm not sure what are you trying to do, but I suspect your problem is in hangar.bank. It contains some hangar music events. It is not convenient, I know, I'll fix it in near future.

**Comment by RadonSound** [comment]
@OXEPU3M menu and new_menu are the events that plays in login screen.
for the 8 nations hangar theme, you can simply type " main " in the search bar and every event for the hangarmusic pops up
![Screenshot (568)](https://user-images.githubusercontent.com/67481432/104305151-9f715b80-54cc-11eb-8566-3a3c37f5faea.png)
so if you only want to use 1 track for all you can simply copy&paste your track into the main themes
And don't forget add loop region (blue bar on the top)

**Comment by OXEPU3M** [comment]
@RadonSound yes, Im talking about this. I put samples  in new_menu event, but loading screen played standart music.
And about "main themes". When you switch to nation it starts to play national main theme. If I just copy sample it plays with break and silence between them.
I want to make a single loop for every menu part and every nation without breaks, like new_menu theme which plays from login screen.
When you just login, you can switch nations and music wont change. But when track ends or you came out of battle game uses differ logic with national themes.
Hope you understand this.
Any way to make it use "login screen" logic?

**Comment by RadonSound** [comment]
![Screenshot (569)](https://user-images.githubusercontent.com/67481432/104313326-27a92e00-54d8-11eb-9954-a18155287f82.png)
about the menu (login) theme did you made sure to build these 4 banks? (hangar, aircraft_music, aircraft_music_2, aircraft_music_3) 

about the main hangar themes, im not sure if this is possible because the default game uses different nation themes
